src/Php81/Resources/stubs/CURLStringFile.php

```diff
@@ -0,0 +1,43 @@
+<?php
+
+if (extension_loaded('curl') && class_exists(CURLFile::class, false)) {
+    // https://php.watch/versions/8.1/CURLStringFile#CURLStringFile-polyfill
+    class CURLStringFile extends CURLFile
+    {
+        private $rawData;
+
+        public function __construct(string $data, string $postname, string $mime = 'application/octet-stream')
+        {
+            $this->rawData = $data;
+            parent::__construct('data://' . $mime . ';base64,' . base64_encode($data), $mime, $postname);
```

Comment:
can you try putting always the same mime-type here (`application/octet-stream`)? Does Curl complain if this is out of sync with `$mime`?
because if it does, we'll also have to sync $nale when $mime changes - I hope we won't have to.

Reply:
what about this?